src/routes/service.ts

```diff
@@ -28,7 +28,7 @@ router.get('/getColorScheme', async function (
     response.status(500).send('Error getting color scheme: ' + e);
   } finally {
     if (tempFilename) {
-      await scheduleTemporaryFileDeletion(tempFilename);
+      scheduleTemporaryFileDeletion(tempFilename);
```

Comment:
Quick question, so it looks like the scheduleTemporaryFileDeletion method is async but we are not awaiting it here, is this on purpose?

Reply:
Same question on the usage of this method below too.